src/handlerUtil.js

```diff
@@ -24,36 +27,28 @@ const loggerTypes = {
 };
 
 // The default handler just logs out the message
-const defaultHandler = (messageSet, topic, partition, callback = logger.info) => {
-  messageSet.forEach((m) => {
-    const key = m.message.key.toString(); // logging level
-    const value = JSON.parse(m.message.value.toString());
-    const log = {
-      application: topic,
-      messageTime: value.messageTime,
-      message: value.message,
-    };
-    if (loggerTypes[key]) {
-      loggerTypes[key](log);
-    } else {
-      callback('Logging with unknown key');
-      logger.info(log);
+const loggerHandler = (messageSet, topic, partition, callback = logger.info) => {
```

Comment:
Can you add param documentation for this function? Is messageSet a promise?

Reply:
Thanks. So messageSet isn't a promise? Then why are you using Promise.each?

Reply:
I just went with the example given [here:](https://www.npmjs.com/package/no-kafka#groupconsumer-new-unified-consumer-api)

And I discovered it is an array from the documentation (screenshot below)

<img width="1440" alt="Screen Shot 2019-07-31 at 2 19 58 PM" src="https://user-images.githubusercontent.com/24924707/62248995-48da9500-b39e-11e9-9dda-cfd352af5905.png">


(Note: Kafka.Message is just a simple object that is being used the way it is in the function)

Reply:
It seems bluebird Promise accepts any iterable.

Reply:
In that example, they are returning consumer.commitOffset(), which I assume returns a promise.
Here, we are logging synchronously, so using Promise.each isn't necessary.
(Though since it seems no-kafka is expecting the logging to be handled asynchronously it might be worth reconsidering if this is the best way of doing it.)